### PR TITLE
MIC-427: E2E Tests for remaining referral pages (minus CYA)

### PR DESCRIPTION
**Pull request by JackReeveMoJ**

### CODE REVIEW

File: server/routes/journeys/referral/proactive-or-reactive/tests.cy.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  const getContinueButton = () => cy.findByRole('button', { name: /Continue/ })
+  const getIsProactiveReferral = () => cy.findByRole('radio', { name: 'Proactive' })
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubComponents')
+    cy.task('stubGetPrisonerImage')
+    cy.task('stubGetPrisoner')
+
+    cy.task('stubIntervieweeRoles')
```
Comment (bill-moj) [inline server/routes/journeys/referral/proactive-or-reactive/tests.cy.ts:18]: do we need this stub?

File: server/routes/journeys/referral/proactive-or-reactive/tests.cy.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  }
+
+  const validateErrorsMandatory = () => {
+    getContinueButton().click()
+
+    cy.findByRole('link', { name: /Select if this referral is proactive or reactive/i })
+      .should('be.visible')
+      .click()
+    getIsProactiveReferral().should('be.focused')
+    cy.findAllByText('Select if this referral is proactive or reactive').should('have.length', 2)
```
Comment (bill-moj) [inline server/routes/journeys/referral/proactive-or-reactive/tests.cy.ts:63]: `findbyRole('link', ...)` above already asserted that the error message is there. not sure whether we need to repeat it.

**Review by bill-moj — APPROVED** [review]

File: server/routes/journeys/referral/proactive-or-reactive/tests.cy.ts (hunk 1 above)
Comment (JackReeveMoJ) [reply server/routes/journeys/referral/proactive-or-reactive/tests.cy.ts:18]: we do not, good spot :)

File: server/routes/journeys/referral/proactive-or-reactive/tests.cy.ts (hunk 2 above)
Comment (JackReeveMoJ) [reply server/routes/journeys/referral/proactive-or-reactive/tests.cy.ts:63]: We're testing here that the text shows up twice in the DOM, one will be in the error summary box as a link and the other will be next to the input itself

**Review by bill-moj — APPROVED** [review]

**Pull request merged**